service/src/test/java/bio/terra/workspace/service/privateresource/PrivateResourceCleanupServiceTest.java

```diff
@@ -47,8 +47,9 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ActiveProfiles;
 
-// Use application configuration profile in addition to the standard connected test profile.
-@ActiveProfiles({"connected-test", "app-test"})
+// Use application configuration profile in addition to the standard connected test profile
+// inherited from the base class.
+@ActiveProfiles({"app-test"})
```

Comment:
I was copying this and realized the `connected-test` profile is already inherited via BaseConnectedTest